test/amperize.test.js

```diff
@@ -177,6 +177,30 @@ describe('Amperize', function () {
             });
         });
 
+        it('transforms <img> into <amp-img> when no file extension is given', function (done) {
+            // This test is mocked, but works with this specific example.
+            // You can comment out the mocks and the test should still pass.
+            sizeOfMock = nock('https://www.zomato.com')
+                .get('/logo/18163505/minilogo')
```

Comment:
Could you add the match headers assertion here as well? 👍

Reply:
Done!